packages/world/src/schemas/Address.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { console } from "forge-std/console.sol";
+import { IStore } from "store/IStore.sol";
+import { StoreSwitch } from "store/StoreSwitch.sol";
+import { StoreCore } from "store/StoreCore.sol";
+import { SchemaType } from "store/Types.sol";
+import { Bytes } from "store/Bytes.sol";
+import { Schema, SchemaLib } from "store/Schema.sol";
+import { PackedCounter, PackedCounterLib } from "store/PackedCounter.sol";
+
+// -- User defined schema and tableId --
+struct AddressSchema {
+  address addr;
+}
+
+// -- Autogenerated library to interact with tables with this schema --
+// TODO: autogenerate
+
+library AddressSchemaLib {
+  /** Get the table's schema */
+  function getSchema() internal pure returns (Schema schema) {
+    schema = SchemaLib.encode(SchemaType.ADDRESS);
+  }
+
+  /** Register the table's schema */
+  function registerSchema(uint256 tableId) internal {
+    StoreSwitch.registerSchema(tableId, getSchema());
+  }
+
+  function registerSchema(uint256 tableId, IStore store) internal {
+    store.registerSchema(tableId, getSchema());
+  }
+
+  /** Set the table's data */
+  function set(
+    uint256 tableId,
+    bytes32 key,
+    address addr
+  ) internal {
+    bytes32[] memory keyTuple = new bytes32[](1);
+    keyTuple[0] = key;
+    StoreSwitch.setField(tableId, keyTuple, 0, abi.encodePacked(addr));
+  }
+
+  /** Get the table's data */
+  function get(uint256 tableId, bytes32 key) internal view returns (address) {
+    bytes32[] memory keyTuple = new bytes32[](1);
+    keyTuple[0] = key;
+    bytes memory blob = StoreSwitch.getRecord(tableId, keyTuple);
+    return address(bytes20(blob));
+  }
+
+  function get(
+    uint256 tableId,
+    IStore store,
```

Comment:
I agree store, table, key would be more intuitive here. The order is mostly because `tableId` is the argument that is "removed" by the table libraries when they wrap the schema libraries, so it's the first argument of all the functions in the schema library. Other methods except from this one don't have a `store` reference (because it's implicitly either `this` or `msg.sender` via `StoreSwitch`) and it feels weird to remove the first argument in the table libraries for every function except this one where we'd remove the second argument.